### ISSUE TITLE
Lambda failing 

### ISSUE THREAD

**Issue by baptiste-s** [druids/terraform-aws-asg-instance-refresh#8]
Hi, 

I use the module as is but I am getting error from the lambda:

```
{
  "errorMessage": "Expecting value: line 1 column 1 (char 0)",
  "errorType": "JSONDecodeError",
  "stackTrace": [
    "  File \"/var/task/lambda.py\", line 179, in handler\n    return main()\n",
    "  File \"/var/task/lambda.py\", line 156, in main\n    image_id = get_current_image_id()\n",
    "  File \"/var/task/lambda.py\", line 57, in get_current_image_id\n    param_value = json.loads(param['Parameter']['Value'])\n",
    "  File \"/var/lang/lib/python3.8/json/__init__.py\", line 357, in loads\n    return _default_decoder.decode(s)\n",
    "  File \"/var/lang/lib/python3.8/json/decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n",
    "  File \"/var/lang/lib/python3.8/json/decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\n"
  ]
}
```

looks like it cannot get the imageId from the json for some reason : 
    `param_value = json.loads(param['Parameter']['Value'])`

here is my tf code : 

```
module "asg-instance-refresh" {
  source                 = "radimsuckr/asg-instance-refresh/aws"
  version                = "0.2.0"
  autoscaling_group_name = aws_autoscaling_group.BastionInstance.name
  launch_template_name   = aws_launch_template.BastionInstance.name
  ami_ssm_parameter      = "/aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2"
}
```

thx for helping :)

**Comment by trebidav** [comment]
what is your region?

**Comment by baptiste-s** [comment]
eu-west-1

**Comment by radimsuckr** [comment]
Hi @baptiste-s,

I found the issue. It happens because the SSM parameter that you specify has a completely different value than the default parameter that I expect. See examples below.

`/aws/service/ecs/optimized-ami/amazon-linux-2/recommended` contains:

```json
{"ecs_agent_version":"1.60.1","ecs_runtime_version":"Docker version 20.10.7","image_id":"ami-0bb7c2ee184bc8249","image_name":"amzn2-ami-ecs-hvm-2.0.20220328-x86_64-ebs","image_version":"2.0.20220328","os":"Amazon Linux 2","schema_version":1,"source_image_name":"amzn2-ami-minimal-hvm-2.0.20220316.0-x86_64-ebs"}
```

Whereas your `/aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2` contains just:

```
ami-010c802cb2f348b54
```

Sadly I currently don't have spare time for "fixing" this (or rather for making it more universal) but I'd be more than happy to review and accept a PR for this! :slightly_smiling_face:

**Comment by baptiste-s** [comment]
thank you for your time. 

I am not very good at python but I submited a PR.

**Comment by radimsuckr** [comment]
Resolved in #9
